pkg/utils/k8sutil.go

```diff
@@ -210,6 +210,9 @@ func EnsureK8sResources(ns, name string, funcObj *spec.Function, client kubernet
 	labels := map[string]string{
 		"function": name,
 	}
+	for k, v := range funcObj.Metadata.Labels {
```

Comment:
My mistake. The map will update the `function` key to new value so there is no duplicate. But we should decide if we want to allow that.